cfme/configure/configuration/region_settings.py

```diff
@@ -1038,21 +1038,31 @@ def set_replication(self, updates=None, replication_type=None, reset=False, vali
                 'password': (
                     updates.get('password') if updates.get('password') else db_creds.password)
             })
-        else:
+        elif replication_type == 'remote':
             view = navigate_to(self, 'RemoteAdd')
             # TODO fill remote settings will be done after widget added
         if reset:
             view.reset_button.click()
             view.flash.assert_message('All changes have been reset')
         else:
             try:
-                if validate:
-                    view.action_dropdown.item_select("Validate")
-                    view.flash.assert_success_message(
-                        "Subscription Credentials validated successfully"
-                    )
-                view.accept_button.click()
-                view.save_button.click()
+                if replication_type == 'global':
+                    if validate:
+                        view.action_dropdown.item_select("Validate")
+                        view.flash.assert_success_message(
+                            "Subscription Credentials validated successfully"
+                        )
+                    view.accept_button.click()
+                    view.save_button.click()
+                    view.flash.assert_success_message('Replication configuration save initiated.'
```

Comment:
We do not want the flash messages here in view. We can do that assertion in the test cases if required.